start.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Start script for disqualified-officers-delta-consumer
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
+    source ~/.chs_env/private_env
+    source ~/.chs_env/global_env
+    source ~/.chs_env/disqualified-officers-delta-consumer/env
+
+    PORT="${DISQUALIFIED_OFFICERS_DATA_API_PORT}"
```

Comment:
`DISQUALIFIED_OFFICERS_DELTA_CONSUMER_PORT`?

Reply:
Good spot, I'll change that now